modules/prebidServerBidAdapter/index.js

```diff
@@ -593,7 +593,9 @@ function emitNonBids(seatnonbid, auctionId) {
  * @param {array} newEidPermissions
  */
 function setEidPermissions(newEidPermissions) {
```

Comment:
I don't think this would cause an empty user.ext.eids. This only affects `ext.prebid.data.eidpermissions` and is already [guarded against empty values](https://github.com/prebid/Prebid.js/blob/de228261a0e5eae6976168299f5e41485481db9e/modules/prebidServerBidAdapter/ortbConverter.js#L62-L69).